rpc/flipt/evaluation/evaluation.proto

```diff
@@ -232,7 +233,7 @@ message EvaluationNamespaceSnapshotRequest {
   string reference = 2;
 }
 
-// flipt:sdk:ignore
 service DataService {
+  option (google.api.api_visibility) = {restriction: "internal"};
```

Comment:
we should probably make this public as far as OpenAPI docs go. although we still don't want to generate the SDK methods for it (GRPC  / REST)